config/serverless.chat.yml

```diff
@@ -1,5 +1,68 @@
 # Chat Project Config
 
+## Fine-tune the chat solution feature privileges. Also, refer to `serverless.yml` for the project-agnostic overrides.
```

Comment:
Maybe in this comment we can also call out that these feature privilege overrides are set individually for each project type?